asset_dashboard/models.py

```diff
@@ -149,9 +149,19 @@ class ProjectCategory(models.Model):
 
     category = models.TextField(null=False)
     subcategory = models.TextField(null=True)
+    slug = models.SlugField(max_length=250, null=True, blank=True)
```

Comment:
Couple things.

I led you astray when I called this a slug. Within django world, a slug is something that should be okay to appear in a url and so typically "consists of only letters, numbers, underscores or hyphens".

So sorry for that. Let's just call this "name" and have this be a "TextField"

Also, this should not be nullable or blankable?

Reply:
I made it null/blank because I thought that an admin user would try and save without populating the slug. 

I worry about an admin user messing with that field, whenever they're adding categories within the admin section. Is there a way to prevent that from happening? Or, is there a way to add a message to that admin form that tells them what the field is for?

Here is the requested change: 106093f.

Reply:
Ignore that last bit. I excluded the field from the admin: 54c2145.